code/game/machinery/syndicatebomb.dm

```diff
@@ -399,6 +399,12 @@
 	range_light = 20
 	range_flame = 20
 
+// Special bomb core for inside the lavaland syndicate base walls. Players should not be able to interact with this.
+/obj/item/bombcore/large/syndicate_base
+	installed = TRUE
+	anchored = TRUE
+	invisibility = INVISIBILITY_OBSERVER
```

Comment:
Is this actually needed if the bombcore is force moved inside the wall anyway?

Reply:
Yes, because it does still appear in the context menu.